inst/report/summary_report.Rmd

```diff
@@ -6,53 +6,40 @@ params:
   output_zip: NA
   spectrum_file: NA
   options: NA
+  lang: NA
 output:
   html_document:
+    css: styles.css
     includes:
       in_header: header.html
     anchor_sections: false
 ---
 
-``` {css, echo = FALSE}
+::: {#translate lang="en"}
```

Comment:
Use `:::` custom blocks https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html for managing inclusion of text